.travis/create_utplsql_user.sh

```diff
@@ -4,5 +4,15 @@ set -ev
 #create user
 "$ORACLE_HOME/bin/sqlplus" -L -S / AS SYSDBA <<SQL
 set echo off
-@@$(dirname "$(readlink -f "$0")")/create_utplsql_user.sql $UT3_USER $UT3_PASSWORD $UT3_TABLESPACE
+@@$(dirname "$(readlink -f "$0")")/create_utplsql_owner.sql $UT3_USER $UT3_USER_PASSWORD $UT3_USER_TABLESPACE
+SQL
+#give grants
+"$ORACLE_HOME/bin/sqlplus" -L -S $UT3_OWNER/$UT3_OWNER_PASSWORD <<SQL
+set echo off
+@@$(dirname "$(readlink -f "$0")")/utplsql_grants.sql $UT3_USER 
+SQL
+# create synonyms
+"$ORACLE_HOME/bin/sqlplus" -L -S / as sysdba <<SQL
+set echo off
+@@$(dirname "$(readlink -f "$0")")/utplsql_synonyms.sql $UT3_OWNER "$UT3_USER." ""
```

Comment:
Can you make sure that we can simply pass `$UT3_USER` without "dot" as a parameter?